request-manager/graph/resolver.go

```diff
@@ -412,23 +390,24 @@ func (o *resolver) buildSequence(cfg buildSequenceConfig) (*Graph, error) {
 	return reqGraph, nil
```

Comment:
Time to clean up part of code you didn't change but needs to be clarified, too. Above, the block that starts with,
```go
// Mark all vertices in sequence except start vertex start sequence id
```
It seems like we can and should be assigning sequence IDs while we're building each sequence? Also, the comment talks about, "A graph is built by constructing its inner most components, ..." which doesn't make sense in its own context, especially since the code block here isn't recursing or building anything.

Reply:
If we want to set sequence IDs while building it, we need to either 1) pass it to `buildSingleVertexGraph` and `newReqGraph`, and have the latter change its behavior based on use case (we're about to build a new request subgraph, or we're just using it to wrap expanded sequences); or 2) just set it all ourselves in `buildSequence` as we go.

I think it's better the way it is right now.

I'm not sure what's up with the comment--for one, it certainly seems to be setting `seqId` for the source and sink. For another, the comment initially says we don't set it for source, then says we don't set it for source or sink. And I have no idea what that TODO's about.

I remember being super confused by this comment when I was first reading the code, and just sort of hoping it'd make sense once I worked with the code more. It still makes no sense to me, so if you have any enlightening information, please share. Otherwise, I'll just update the comment to describe what I think is happening.

Reply:
If it's not trivial to roll the seq ID into the nodes/req subgraphs while we're building them, then yes: let's just remove and rewrite the comment so it's correct and helpful.